### ISSUE TITLE
[Feature Request] More quick response

### ISSUE THREAD

**Issue by ido1990** [GravityBox/GravityBox#1593]
Hi,
Could you add an option to add more quick response messages on the dialer?
I hope it can be done with GraviryBox
Thanks!

**Comment by C3C0** [comment]
Dialer became something that is not bound to system so tightly as it was in the past. It became more independent and nowadays Google is able to provide independent updates via Play Store. Moreover they started to obfuscate it which makes reverse engineering far more difficult and has negative impact on overall GravityBox project manageability. GravityBox's policy is to provide system level tweaks not touching 3rd party apps. The only tweaks that touched 3rd party app were launcher tweaks (for Google Launcher) and it didn't proof as good as I needed to adjust code with every launcher update.
TLDR; I am not planning further Dialer tweaks due to its nature becoming a "3rd party app". I even already reworked all existing logic that could be transferred to system level (within Phone tweaks section) to minimize dependency (as can be seen here https://github.com/GravityBox/GravityBox/commit/6c1413b7a7b5de9101de215021e676b690d8abb1 )
For Dialer, it would rather be more suitable to have dedicated xposed module.

**Comment by ido1990** [comment]
@C3C0 
I did some research about it and I found that the Quick Responses is on "com.android.server.telecom" (QuickResponseUtils) and not in the dialer app.
Does it make my request eligible?
Thanks!

**Comment by C3C0** [comment]
Will check
